src/js/components/FormField/FormField.js

```diff
@@ -106,10 +106,13 @@ class FormFieldContent extends Component {
       onBlur,
       onFocus,
       margin,
+      underline,
```

Comment:
I'm not sure about the naming of the prop, doesn't seem consistent with grommet props.
A suggestion, let's try to get a `border` prop and if it is "none" than the underline will be removed, or any other given border on that prop will be applied correspondingly. thoughts?

Reply:
Agreed. We should do something with a `border` property.

Reply:
Okay, this feedback makes sense. I will work on doing a `border` property.

Reply:
this looks similar to this https://github.com/grommet/grommet/pull/3096 maybe we can finish that PR?

Reply:
@oorestisime You're right. I think this PR would solve the issues noted in that PR.